src/main/java/Elements/AbstractBaseElement.java

```diff
@@ -0,0 +1,70 @@
+package Elements;
+
+import Utils.InstanceDriver;
+import Utils.DriverWait;
+import org.openqa.selenium.By;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public abstract class AbstractBaseElement {
+
+    private final By element;
```

Comment:
use such approach

protected final Wait wait;
private final WebDriver driver;
private final By element;

public AbstractElement(final WebDriver driver, final By locator) {
  this.driver = driver;
  this.locator = locator;
  wait = new Wait(driver, locator);
}